### DISCUSSION TITLE
Link download attribute doesn't work

### DISCUSSION THREAD

**Discussion by Glecun** [alpinejs/alpine#1966] [7. Bugs (archived)] [answered]
Hi,

I think the attribute 'download' doesn't work on link tag (\<a\>) when we inject the href
```
<head>
  <script src="//unpkg.com/alpinejs" defer></script>
  <script>
  function component() {
    return {
      href:
        'https://www.google.fr/images/branding/googlelogo/2x/googlelogo_color_272x92dp.png'
    };
  }
  </script>
</head>

<div x-data="component()">
  <a :href="href" download>download</a>
</div>
```

When I click on the link, I expect the navigator to directly download the image but it redirects me instead.
[example on stackblitz](
https://stackblitz.com/edit/js-kgozwc?embed=1&file=index.html)

**Comment by MuzafferDede** [comment]
Does it work without Alpine?

**Comment by SimoTod** [comment, marked as answer]
It's not Alpine, your code is not working regardless. Most browsers won't download the file if it's a cross-origin request for security reasons.
